falcon/media/validators/jsonschema.py

```diff
@@ -0,0 +1,47 @@
+from __future__ import absolute_import
+
+import falcon
+
+try:
+    import jsonschema
+except ImportError:
+    pass
+
+
+def validate(schema):
+    """Decorator that validates ``req.media`` using JSONSchema
+
+    Args:
+        schema (dict): A dictionary that follows the JSONSchema specification.
+            See `json-schema.org <http://json-schema.org/>`_ for more
+            information on defining a compatible dictionary.
+
+    Example:
+        .. code:: python
+
+            from falcon.media.validators import jsonschema
+
+            # -- snip --
```

Comment:
I like the way you use these `# -- snip --` comments. I'm so totally going to steal this. (You know, because I'm a great artist.) 💯